src/main/groovy/io/github/cosmicsilence/scalafix/ScalaSourceSet.groovy

```diff
@@ -20,8 +20,8 @@ class ScalaSourceSet {
         return sourceSet.name
     }
 
-    List<File> getClassesDirs() {
-        return sourceSet.output.classesDirs.toList()
+    List<File> getFullClasspath() {
+        return (sourceSet.output.classesDirs + sourceSet.compileClasspath).toList()
```

Comment:
the plugin was not including the dependencies (jars) in the classpath passed to Scalafix.